pipeline/transformer/transformer_sierra/src/test/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraTransformerTest.scala

```diff
@@ -992,6 +992,47 @@ class SierraTransformerTest
     }
   }
 
+  it("includes items based on the order records") {
```

Comment:
Worth testing some invalid cases?

Reply:
Was there anything you think I've missed?

Reply:
Ah hang on, github was helpfully hiding the "large diff" on the tests file 🙃